week-03/README.md

```diff
@@ -16,69 +16,72 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Textbook Chapter 3, Problem 1]
-
+- Question (lsun20-stat6250): Why the RUN statements can in the column one? Is there any example?
 
 
 [Course Textbook Chapter 3, Problem 2]
+- Question (lsun20-stat6250): which window will show your error specifically?
 
-
+- Answer (lsun20-stat6250): the log window will display the messages about the error.
 
 [Course Textbook Chapter 3, Problem 3]
-
+- Question (lsun20-stat6250): what is the maxmum length of a quoted string?
 
  
 [Course Textbook Chapter 3, Problem 4]
+- Question (lsun20-stat6250): Which errors except Syntax error also will cause SAS to stop processing the step in which the error occurred?
 
 
-
-[Course Textbook Chapter 3, Problem 5]
-
+[Course Textbook Chapter 3, Problem 5] 
```

Comment:
Existing content edited; please remove spaces added after problem headers (after the closing square brace) and DM me in Slack with the PR URL for re-evaluation